sound/soc/sof/pm.c

```diff
@@ -422,3 +438,41 @@ int snd_sof_suspend(struct device *dev)
 	return sof_suspend(dev, false);
 }
 EXPORT_SYMBOL(snd_sof_suspend);
+
+int snd_sof_set_dsp_power_state(struct snd_sof_dev *sdev,
```

Comment:
@keyonjie I dont see this being used anyehere yet. Is it coming?

Reply:
yes, it is coming, in my next PR. :)
Still have 2 PRs after this done.